alibi/explainers/tests/conftest.py

```diff
@@ -8,18 +8,46 @@
 from alibi.explainers import ALE
 from alibi.explainers import AnchorTabular
 from alibi.explainers import KernelShap, TreeShap
-from alibi.explainers.tests.utils import predict_fcn, adult_dataset, iris_dataset, boston_dataset, MockTreeExplainer
+from alibi.explainers.tests.utils import predict_fcn, adult_dataset, iris_dataset, boston_dataset, mnist_dataset, \
+    MockTreeExplainer
 from alibi.tests.utils import MockPredictor
-from tensorflow.keras.layers import Conv2D, Dense, Dropout, Flatten, MaxPooling2D, Input
-from tensorflow.keras.models import Model
+import tensorflow as tf
 from sklearn.ensemble import RandomForestClassifier
 
+import alibi_test_models
+
 
 # A file containing fixtures that can be used across tests
 
 # Fixtures that return datasets can be combined with classifier
 # fixtures to generate models for testing.
 
+@pytest.fixture
+def model(request):
+    """
+    This fixture loads a pre-trained test model by name from the
+    alibi-test-models helper package.
+    """
+    name = request.param
+    return alibi_test_models.load(name)
+
+
+@pytest.fixture
+def models(request):
+    """
+    This fixture loads multiple pre-trained test-models by name from the
+    alibi-test-models helper package.
+    """
+    models = []
+    for name in request.param:
+        models.append(alibi_test_models.load(name))
+    return models
```

Comment:
Possibly, but that would make the single-model fixture dependency slightly annoying (having to index into [0]). Another alternative is to validate request and return a list if it a list, otherwise a model if it is a string.